hub/jupyterhub_config.py

```diff
@@ -13,6 +13,28 @@
 # Our simplest user image! Optimized to just... start, and be small!
 c.KubeSpawner.singleuser_image_spec = 'yuvipanda/simple-singleuser:v1'
 
+# Configure dynamically provisioning pvc
+c.KubeSpawner.pvc_name_template = 'claim-{username}-{userid}'
+c.KubeSpawner.storage_class = 'single-user-storage'
+c.KubeSpawner.access_modes = ['ReadWriteOnce']
+c.KubeSpawner.storage = '10GiB'
+
+# Add volumes to singleuser pods
+c.KubeSpawner.volumes = [
```

Comment:
When a pod os launch, it needs to know which PVC to attach to. Here we specify to attach it to the claim with ```'claim-{username}-{userid}'```